src/Arcus.WebApi.Security/Authorization/Extensions/FilterCollectionExtensions.cs

```diff
@@ -23,13 +27,14 @@ public static FilterCollection AddJwtTokenAuthorization(this FilterCollection fi
         /// </summary>
         /// <param name="filters">All filters that are being applied to the request pipeline</param>
         /// <param name="configureOptions">Configuration options for using JWT token authorization</param>
+        /// <param name="claimCheck">Custom claims key-value pair to validate against</param>
         public static FilterCollection AddJwtTokenAuthorization(
             this FilterCollection filters, 
-            Action<JwtTokenAuthorizationOptions> configureOptions)
+            Action<JwtTokenAuthorizationOptions> configureOptions, IDictionary<string, string> claimCheck = null)
```

Comment:
We may consider creating an overload instead of an optional parameter because it will result in `MethodNotFound` exceptions when the packages are updated seperatly.

Reply:
done